proposals/0003-health-checks.md

```diff
@@ -0,0 +1,103 @@
+# Service Healthchecks
+
+* Proposal: [MP-0003](0003-health-checks.md)
+* Authors: [Clement Escoffier](https://github.com/cescoffier), [Heiko Rupp](https://github.com/pilhuhn), [Heiko Braun] (https://github.com/heiko-braun)
+
+* Status: **Awaiting review**
+
+*During the review process, add the following fields as needed:*
+
+* Decision Notes: [Discussion thread topic covering the  Rationale](https://groups.google.com/forum/#!forum/microprofile), [Discussion thread topic with additional Commentary](https://groups.google.com/forum/#!forum/microprofile)
```

Comment:
The links point to the Google group only. Could you link to the specific threads instead? Or possibly even better put the contents in here, so that one has all the information in one place (specifically, it may be hard to reason what's the agreed on outcome from a lengthy email thread).